pxr/usd/usd/testenv/testUsdValidationRegistry.cpp

```diff
@@ -310,6 +294,27 @@ void TestUsdValidationRegistry()
     }
 }
 
+void
+TestUsdValidators()
+{
+    UsdValidationRegistry& registry = UsdValidationRegistry::GetInstance();
+
+    // The following test keeps track of all the available validators within
+    // UsdCoreValidators keyword, hence as new validators are added under
+    // this keyword this unit test will have to be updated.
+    const UsdValidatorMetadataVector coreValidatorMetadata =
+            registry.GetValidatorMetadataForKeyword(
+                    UsdValidatorKeywordTokens->UsdCoreValidators);
+    TF_AXIOM(coreValidatorMetadata.size() == 1);
+    const std::vector<TfToken> expectedValidatorNames =
+            {UsdValidatorNameTokens->compositionErrorTest,
+             UsdValidatorNameTokens->stageMetadataChecker};
```

Comment:
Thanks. But since you are touching this code, can I request you to update this to the subset check instead of equality check (Similar to this https://github.com/PixarAnimationStudios/OpenUSD/blob/dev/pxr/usd/usdShade/testenv/testUsdShadeValidators.cpp#L22-L46).

Reasoning being, if a client is running this test in their own environment and they have validators available in their plugins which are listing "UsdCoreValidators" as a keyword, the equality will fail, so all we can do here is to make sure the validator provides by openUSD under UsdCoreValidators keyword are found.

Reply:
And you mentioned the following assert is failing for you?

`TF_AXIOM(!PlugRegistry::GetInstance().RegisterPlugins(testDir).empty());`

- This could be because the "pxr/usd/usd/testenv/testUsdValidationRegistry" dir is not being installed property for you .. I don't know why that will be though.
- Are you running the test via ctest? 
- The test uses [ArchGetCwd()](https://github.com/PixarAnimationStudios/OpenUSD/blob/be2078039b0c3fa9f9f4a7b8e48759266029609a/pxr/usd/usd/testenv/testUsdValidationRegistry.cpp#L322C33-L322C43), which should have returned the path where testUsdValidationRegistry is installed.

I checked all build configurations internally and this test does pass in all.

Please let me know if you still need help with this.

Reply:
updated to do the subset check rather than equality, sorry i missed that the first time around.

following up on slack about testing setups